### ISSUE TITLE
Image refresh for fedora-atomic

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/cockpit#7410]
Image refresh for fedora-atomic
 * [ ] FAIL: image-refresh fedora-atomic

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tests-p2npt.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-7410-20170731-064444/

**Comment by cockpituous** [comment]
Task failed: http://fedorapeople.org/groups/cockpit/logs/image-refresh-7410-20170731-064444/

**Comment by stefwalter** [comment]
This code is breaking:

```
    # HACK - Find a better way to compute the ref.
    # https://lists.projectatomic.io/projectatomic-archives/atomic-devel/2016-July/msg00015.html

    remote=$(ostree remote list)
    ref=$(python -c 'import json; print json.load(open("/usr/share/rpm-ostree/treefile.json"))["ref"]')
    checkout="$remote:$ref"
```

**Comment by stefwalter** [comment]
Any ideas of how to solve this?

**Comment by petervo** [comment]
Fixed in b6747182df3f63e32a47bf42238eaae6d47e4546 @petervo petervo committed with dperpeet 17 days ago
